packages/debian/metalk8s-sosreport-deb-2.4.0/debian/metalk8s-sosreport-deb/usr/share/sosreport/sos/plugins/containerd.py

```diff
@@ -0,0 +1,73 @@
+#! /bin/env python
```

Comment:
I guess you did a `git add -A` some files are generated on build time and do not need to be in the cvs